packages/react/src/components/LocaleModal/LocaleModal.js

```diff
@@ -162,6 +168,17 @@ const LocaleModal = ({ isOpen, setIsOpen }) => {
   function close() {
     setIsOpen(false);
   }
+
+  /**
+   * @private
+   *
+   * @param {number} keyCode - The code of the keyboard key pressed.
+   */
+  function _closeFilterOnEscapeKeyPress({ keyCode }) {
```

Comment:
Does this need to be passed in as an object?

Reply:
The `keyCode` property came from the keydown event, which is an object.

Reply:
ok. For JSDocs, we'd want to probably document that then:

```
/**
 * Detects for the esc key press then closes the modal
 * 
 * @param {object} event Event object from the event listener
 * @param {number} event.keyCode code of the keyboard key pressed
 * @private
 */
```

Reply:
Thanks, Jeff!